recipe/meta.yaml

```
@@ -20,8 +20,8 @@ requirements:
     - python >=3.6
     - setuptools
   run:
-    - h11 >=0.8,<0.10
-    - h2 ==3.*
+    - h11 ==0.*
+    - h2 >=3,<=5
```

https://github.com/encode/httpcore/blob/84c7738de7fd23ec20187964a33b5b9a0de01493/setup.py#L56-L58 shows this should be:

```suggestion
    - h2 >=3,<5
```